README.md

```diff
@@ -50,7 +50,9 @@ worker: bundle exec sidekiq -C ./config/sidekiq.yml
 ### 4. Configure puppet
 
 - Set `REDIS_HOST` and `REDIS_PORT` variables. `GOVUK_APP_NAME` should also be
-set, but this is already done by the default `govuk::app::config`.
+set, but this is already done by the default `govuk::app::config`. If you redis instance
```

Comment:
Few typos: "if you**r** Redis instance requires more advance**d** connection settings".